### PR TITLE
Delete account

### PR DESCRIPTION
**Pull request by reshetniktati**
delete user account from db, localstorage

### CODE REVIEW

File: client/src/views/Settings/Tabs/EditPrivacy/index.vue [hunk 1]
```diff
@@ -80,6 +80,8 @@
 </template>
 
 <script>
+// import { setNotifSettings } from '@/services/notification.js'
```
Comment (igorkokotko) [inline client/src/views/Settings/Tabs/EditPrivacy/index.vue]: Why commented?

File: client/src/views/Settings/Tabs/EditPrivacy/index.vue [hunk 2]
```diff
@@ -88,7 +90,18 @@ export default {
...
+    }
+  },
+  methods: {
+    toggle () {
+      // this.$emit('input', !this.value)
+      // console.log(e.target)
+      // console.log(e.target.checked)
+      console.log(!this.value)
+      // this.checked.push(value)
+      console.log(this.checked)
```
Comment (igorkokotko) [inline client/src/views/Settings/Tabs/EditPrivacy/index.vue]: Uselesss method with comments and console?Why this here!?

File: server/controllers/profiles.js [hunk 3]
```diff
@@ -128,6 +128,14 @@ const profileAction = asyncMiddleware(async (req, res) => {
   }
 })
 
+const deleteAccount = asyncMiddleware(async (req, res, next) => {
+  // console.log(req)
+  console.log(req.userId)
```
Comment (igorkokotko) [inline server/controllers/profiles.js]: COnsole! Stop pushing console. Please!

**Review by igorkokotko — CHANGES_REQUESTED** [review]

File: client/src/views/Settings/Tabs/EditPrivacy/index.vue (hunk 1 above)
Comment (reshetniktati) [reply client/src/views/Settings/Tabs/EditPrivacy/index.vue]: It should be in other pr. I pushed unfinished code here by mistake

File: client/src/views/Settings/Tabs/EditPrivacy/index.vue (hunk 2 above)
Comment (reshetniktati) [reply client/src/views/Settings/Tabs/EditPrivacy/index.vue]: It should be in other pr. I pushed unfinished code here by mistake.
I took this code away from this pr

File: server/controllers/profiles.js (hunk 3 above)
Comment (reshetniktati) [reply server/controllers/profiles.js]: console was deleted

File: client/src/views/Settings/Tabs/DeleteAccount/index.vue [hunk 4]
```diff
@@ -12,22 +12,51 @@
...
   data () {
     return {
       confirm: false
     }
+  },
+  methods: {
+    deleteProfile () {
+      let user = this.getProfile
+      deleteAccount(user)
+        .then((res) => {
```
Comment (igorkokotko) [inline client/src/views/Settings/Tabs/DeleteAccount/index.vue:38]: Would be nice to handle errors if such occur

File: client/src/views/Settings/Tabs/DeleteAccount/index.vue [hunk 5]
```diff
@@ -12,22 +12,51 @@
...
+          this.logOut()
+        })
+    },
+    logOut () {
+      this.$store.dispatch('auth/signin', { token: '', user: '' })
```
Comment (igorkokotko) [inline client/src/views/Settings/Tabs/DeleteAccount/index.vue]: Again you doing the direct call from $store, please rewrite to ...mapActions

**Review by igorkokotko — CHANGES_REQUESTED** [review]

File: client/src/views/Settings/Tabs/DeleteAccount/index.vue (hunk 5 above)
Comment (reshetniktati) [reply client/src/views/Settings/Tabs/DeleteAccount/index.vue]: done

File: client/src/views/Settings/Tabs/DeleteAccount/index.vue (hunk 4 above)
Comment (reshetniktati) [reply client/src/views/Settings/Tabs/DeleteAccount/index.vue:38]: done

**Review by igorkokotko — APPROVED** [review]

**Pull request merged**